esql/utils.py

```diff
@@ -0,0 +1,51 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0; you may not use this file except in compliance with the Elastic License
+# 2.0.
+from antlr4 import ParserRuleContext
+from antlr4.tree.Trees import Trees
+from esql.errors import ESQLSemanticError
+# from esql.EsqlBaseParser import EsqlBaseParser
+
+
+# def print_tree(parser: EsqlBaseParser, ctx: ParserRuleContext):
+def print_tree(parser, ctx: ParserRuleContext):
+    """Print the parse tree."""
+    print(Trees.toStringTree(ctx, None, parser))
+
+
+# def pretty_print_tree(ctx: EsqlBaseParser.SingleStatementContext, indent: int = 0, is_last: bool = True):
```

Comment:
Reminder, I think we can remove these commented out function headers and add the type hinting to the active headers? 
```suggestion

```